### DISCUSSION TITLE
ScriptReference/UIElements.BaseListView-itemsRemoved

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#698] [Notes]
# ScriptReference/UIElements.BaseListView-itemsRemoved

https://docs.unity3d.com/ScriptReference/UIElements.BaseListView-itemsRemoved.html

**Comment by poillic** [comment]
This method has a weird behaviour. 
The item is visually removed from the list but is still present in the list.itemSource, you need to remove another item to get the first one removed from the list.itemSource.

With this code :
```cs
list.itemsRemoved += ( e ) => {
    Debug.Log( list.itemsSource.Count );
};
```
`list.itemSource.Count` value is always the number of item display in the list + 1 there is an offset between what we see and the data.
